File: docs/command_interface/src/resources/source/sample.pl
use lib 'C:\\Program Files (x86)\\Apache Software Foundation\\Trafodion Command Interface\\lib\\perl';
use Session;

# create a new session
$sess = Session->new();

# connect to the database
$sess->connect("user1","password","16.123.456.78","23400");

$retval=$sess->execute(" set schema TRAFODION.CI_SAMPLE ");
print $retval;

# Execute sample queries
$retval=$sess->execute("select * from employee"); print $retval;
$retval=$sess->execute("get statistics"); print $retval;

# disconnect from the database
print "\n\nSession 1: Disconnecting first session. \n\n";
$sess->disconnect();

File: docs/command_interface/src/resources/source/sample.py
import os import sys

## Modify this path
sys.path.append("C:\\Program Files (x86)\\Apache Software Foundation\\Trafodion Command Interface\\lib\\python")
import Session

# create a new session
sess = Session.Session()

# Connect to the database
x=sess. connect ("user1","password","16.123.456.78","23400")

# Execute sample queries

# execute takes the query string as argument
setSchema = "set schema TRAFODION.CI_SAMPLE"
selectTable = "select * from employee"
getStats = "get statistics"

#Contruct a list of SQL statements to be executed
queryList = [setSchema, selectTable, getStats] print "\n";

for query in queryList:
   print sess.execute (query)

# disconnect the session
sess.disconnect()
del sess
sess=None
